### ISSUE TITLE
Unable to count() a column with type boolean

### ISSUE THREAD

**Issue by lonwern** [scylladb/scylla#4088]
This is Scylla's bug tracker, to be used for reporting bugs only.
If you have a question about Scylla, and not a bug, please ask it in
our mailing-list at scylladb-dev@googlegroups.com or in our slack channel.

- [x] I have read the disclaimer above, and I am reporting a suspected malfunction in Scylla.

*Installation details*
Scylla version (or git commit hash): 3.0.rc4
Cluster size: 1
OS (RHEL/CentOS/Ubuntu/AWS AMI): CentOS 7

```
cqlsh> create keyspace ks WITH replication = {'class': 'SimpleStrategy', 'replication_factor': 1};
cqlsh> use ks;
cqlsh:ks> create table t (p int, c int, v boolean, primary key (p, c));
cqlsh:ks> select count(v) from t where p = 1 and c = 1;
InvalidRequest: Error from server: code=2200 [Invalid query] message="Invalid call to function count, none of its type signatures match (known type signatures: system.count : (uuid) -> bigint, system.count : (timeuuid) -> bigint, system.count : (timestamp) -> bigint, system.count : (date) -> bigint, system.count : (text) -> bigint, system.count : (double) -> bigint, system.count : (float) -> bigint, system.count : (decimal) -> bigint, system.count : (varint) -> bigint, system.count : (bigint) -> bigint, system.count : (int) -> bigint, system.count : (smallint) -> bigint, system.count : (tinyint) -> bigint)"
```

**Comment by tzach** [comment]
Look similar to https://github.com/scylladb/scylla/issues/3824

**Comment by jwnx** [comment]
Also fixed by 474bb4e

**Comment by avikivity** [comment]
Fixed in 3.1.
